### PR TITLE
Mt improvement 01

### PR DESCRIPTION
**Pull request by yuri-vayne-jala**

### CODE REVIEW

File: IDT.MoneyTransfer/App.config [hunk 1]
```diff
@@ -14,6 +14,8 @@
 		<add key="Browser" value="Chrome" />
 		<!-- The Url of the application under test -->
 		<add key="ApplicationUrl" value="https://feature3.retailers.awsqa.bossrow.net" />
+		<add key="Retailer_User" value="1128683494" />
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/App.config]: I think that you should use Pascal naming convention

File: IDT.MoneyTransfer/Main/UI/Pages/Login/RetailerLoginPage.cs [hunk 2]
```diff
@@ -33,8 +33,8 @@ public class RetailerLoginPage : BasePage, ILoginPage
         /// <param name="userType">User type in enum type.</param>
         public void LoginAsUserType(UserType userType)
         {
-            CommonActions.SendKeys(this.txtUserName, Constants.Username);
-            CommonActions.SendKeys(this.txtPassword, Constants.Password);
+            CommonActions.SendKeys(this.txtUserName, ConfigurationManager.AppSettings["Retailer_User"]);
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/UI/Pages/Login/RetailerLoginPage.cs]: change it using Pascal Naming convention

File: IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs [hunk 3]
```diff
@@ -10,8 +10,9 @@ namespace IDT.MoneyTransfer.Main.Utilities.WebDriver
     using OpenQA.Selenium;
     using OpenQA.Selenium.Interactions;
     using OpenQA.Selenium.Support.UI;
+	using IDT.MoneyTransfer.Main.Resources;
 
-    class WebDriverUtils
+	class WebDriverUtils
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs]: add Doc to the class

File: IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs [hunk 4]
```diff
@@ -10,8 +10,9 @@ namespace IDT.MoneyTransfer.Main.Utilities.WebDriver
...
     {
         public static bool ElementExistsOnpage(string elementName)
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs]: add doc for the method

File: IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs [hunk 5]
```diff
@@ -56,6 +57,11 @@ public static string GetUrl()
             return WebDriverManager.Instance.GetWebDriver.Url;
         }
 
+        public static void GoToUrl(string url)
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs]: add doc for the method

File: IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs [hunk 6]
```diff
@@ -56,6 +57,11 @@ public static string GetUrl()
...
+        {
+            WebDriverManager.Instance.GetWebDriver.Navigate().GoToUrl(url);
+        }
+
         public static void CloseActualWindow()
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs]: add doc for the method

File: IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs [hunk 7]
```diff
@@ -68,9 +74,9 @@ public static void WaitUntil(int seconds)
 
         public static void LogoutPortal()
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs]: add doc for the method

File: IDT.MoneyTransfer/Tests/UI/Hooks/CommonHooks.cs [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ﻿namespace IDT.MoneyTransfer.Tests.UI.Hooks
```
Comment (wMercadoJala) [inline IDT.MoneyTransfer/Tests/UI/Hooks/CommonHooks.cs]: add Header doc

File: IDT.MoneyTransfer/App.config (hunk 1 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/App.config]: Done

File: IDT.MoneyTransfer/Main/UI/Pages/Login/RetailerLoginPage.cs (hunk 2 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/UI/Pages/Login/RetailerLoginPage.cs]: Done

File: IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs (hunk 4 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs]: done

File: IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs (hunk 5 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs]: done

File: IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs (hunk 6 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Main/Utilities/WebDriver/WebDriverUtils.cs]: done

File: IDT.MoneyTransfer/Tests/UI/Hooks/CommonHooks.cs (hunk 8 above)
Comment (yuri-vayne-jala) [reply IDT.MoneyTransfer/Tests/UI/Hooks/CommonHooks.cs]: done

**Pull request merged**